dashboard/app/controllers/reviewable_projects_controller.rb

```diff
@@ -0,0 +1,39 @@
+class ReviewableProjectsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :decrypt_channel_id, only: [:create]
+
+  check_authorization
+  load_and_authorize_resource :reviewable_project, only: [:destroy]
```

Comment:
Realizing that the `:reviewable_project` argument here (which I included in code review comments, my bad) may not be necessary:

https://github.com/CanCanCommunity/cancancan/blob/develop/docs/Authorizing-controller-actions.md#choosing-actions